### PR TITLE
Fix historical Thanksgivings

### PR DESCRIPTION
**Pull request by wtn**

### CODE REVIEW

File: lib/market_open_days/nyse_market_days.csv [hunk 1]
```diff
@@ -1737,11 +1737,11 @@
 2020-11-16
 2020-11-17
 2020-11-18
+2020-11-19
 2020-11-20
 2020-11-23
 2020-11-24
 2020-11-25
-2020-11-26
```
Comment (wakproductions) [inline lib/market_open_days/nyse_market_days.csv:1744]: Bruh these dates are correct!
11/28/2019 - SPY closed at 310.27, was closed on the 28th
11/19/2020 - SPY closed at 357.78, was closed on the 26th

image


image

**Review by wakproductions — CHANGES_REQUESTED** [review]

File: lib/market_open_days/nyse_market_days.csv (hunk 1 above)
Comment (wtn) [reply lib/market_open_days/nyse_market_days.csv:1744]: I agree completely that the market was open on the two days you pointed to on the charts. That's why I added them to the "market open days" file and removed the two instances of Thanksgiving Day that should not have been there.

Thanksgiving

Comment (wakproductions) [reply lib/market_open_days/nyse_market_days.csv:1744]: oh darn that's totally right! what was I thinking?! thanks so much for your contribution!!! sorry for the misunderstanding.

**Review by wakproductions — APPROVED** [review]

**Comment by wtn** [comment]
I fixed a few more historical market holidays. Everything from 2016-01-01 to present should be correct now.

[NYSE Group press release for 2016 & 2017 holidays](https://www.businesswire.com/news/home/20160202006502/en/NYSE-Group-Announces-2017-Holiday-and-Early-Closings-Calendar)

**Review by wakproductions — APPROVED** [review]
lgtm! Thanks!

**Comment by wtn** [comment]
Please merge, thanks.

**Pull request merged**